### PR TITLE
fix: update docs react for better json schema support + remove interval

### PR DESCRIPTION
**Pull request by shanejonas**
image

### CODE REVIEW

File: src/OpenRPCEditor.tsx [hunk 1]
```diff
@@ -28,19 +30,30 @@ const OpenRPCEditor: React.FC<IProps> = (props) => {
         model.dispose();
       }
     };
-  // eslint-disable-next-line react-hooks/exhaustive-deps
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  function handleEditorDidMount(_: any, editor: any) {
+  function handleEditorDidMount(__: any, editor: any) {
```
Comment (zcstarr) [inline src/OpenRPCEditor.tsx:36]: confused by the __ should it be _ ?

File: src/plugins/InspectorPlugin.tsx [hunk 2]
```diff
@@ -23,9 +23,9 @@ const InspectorPlugin: React.FC<IMethodPluginProps> = (props) => {
         openrpcMethodObject: props.openrpcMethodObject,
         request: {
           jsonrpc: "2.0",
-          id: 0,
           method: method.name,
           params: exampleParams || [],
+          id: 0,
```
Comment (zcstarr) [inline src/plugins/InspectorPlugin.tsx:28]: aesthetically more pleasing?

File: src/OpenRPCEditor.tsx (hunk 1 above)
Comment (shanejonas) [reply src/OpenRPCEditor.tsx:36]: `_` is taken by lodash

File: src/plugins/InspectorPlugin.tsx (hunk 2 above)
Comment (shanejonas) [reply src/plugins/InspectorPlugin.tsx:28]: image

inspector does this order for its defaults

Comment (zcstarr) [reply src/plugins/InspectorPlugin.tsx:28]: 👍

File: src/OpenRPCEditor.tsx (hunk 1 above)
Comment (zcstarr) [reply src/OpenRPCEditor.tsx:36]:
```suggestion
  function handleEditorDidMount(_void: any, editor: any) {
```

**Pull request merged**